binary-auth-attestation/src/create-sign-attestion.js

```diff
@@ -0,0 +1,42 @@
+const exec = require('@actions/exec');
+
+// Binauthz attestations gcloud cmd
+const createAttestation = async (
+  artifactUrl, attestor, attestorProject, keyversionProject, keyversionLocation,
+  keyversionKeyring, keyversionKey, keyversion) => {
+  const args = [
+    '--quiet',
+    'beta',
```

Comment:
Do we need to use the beta command? Seems like `gcloud container binauthz` is available also.

Reply:
I can test that

Reply:
sign-and-create is not available for `gcloud container binauthz`